netutil/freeport.go

```diff
@@ -0,0 +1,41 @@
+package netutil
+
+import (
+	"net"
+)
+
+// FreeTCPAddr returns an available TCP port
+func FreeTCPAddr() *net.TCPAddr {
```

Comment:
Too much panic inside this `func`s. I guess that the unique way to trigger them is that OS has no more free ports, isn't it? If that's the case, I'm ok with them.

Reply:
correct. i didn't participate in writing the original package, but that's the explanation i got back then.

Reply:
That's it. Also, this is mainly intended to be used in testing.

If we are not confortable with panic-ing inside the function, maybe we can follow the `Must` pattern we see in the standard library. For example: https://golang.org/src/regexp/regexp.go?s=8678:8714#L227